### DISCUSSION TITLE
Can't drag-and-drop icons on System apps

### DISCUSSION THREAD

**Discussion by tsuliwaensis** [elrumo/macOS_Big_Sur_icons_replacements#1191] [General]
I can't seem to replace icons for system apps like Safari, Messages, and Terminal. I drag the icon to the Get Info sheet, but nothing happens. What am I missing?

**Comment by rodgerwerner** [comment]
I believe this has to do with System Integrity Protection.  You can try disabling that but you can also do something a tiny bit easier.

Back up your original App (Safari for instance).
Make a copy of the App, and you will note that now the name can be changed on the copy, but if you backed up the original and stored it elsewhere (not in the Applications Folder) then you can leave the name alone.
On the copy, use the Get Info method to change the icon.
